submissions/Siusarna/data-structures/server/controllers.js

```diff
@@ -0,0 +1,118 @@
+const responseSender = require('./httpResponce.js');
+
+function isJSON(req) {
+  return req.headers['content-type'] === 'application/json';
+}
+
+function showList(res, list) {
+  const allNodeFromList = list.show();
+  const jsonObjWithAllNode = JSON.stringify({ data: allNodeFromList });
+  responseSender.success(res, jsonObjWithAllNode);
+}
+
+function deleteElementsFromStack(res, stack) {
+  try {
+    const removedValue = stack.pop();
+    const jsonObjWithRemovedValue = JSON.stringify({ data: removedValue });
+    responseSender.success(res, jsonObjWithRemovedValue);
+  } catch (err) {
+    responseSender.error(res, err.message);
+  }
+}
+
+function isValidData(inputData) {
+  return typeof inputData === 'number' || typeof inputData === 'string';
+}
+
+function wrongTypeOfValue(res) {
+  responseSender.error(res, 'wrong type of input value');
+}
+
+function wrongTypeOfContent(res) {
```

Comment:
`wrongTypeOfContent` is a noun. Name of functions should be verb. So better will be `handleWrongTypeOfContent` or `sendWrongTypeOfContentResponse`